miso-web/src/main/webapp/scripts/list.js

```diff
@@ -116,6 +318,12 @@ ListUtils = (function() {
     });
     optionModifier(options, jqTable, errorMessage, columns);
     jqTable.dataTable(options);
+    if (target.hasOwnProperty("searchTermSelector")) {
+      var searchDivId = elementId + '_filter';
+
+      tooltipId = makeSearchTooltip(searchDivId, target);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "tooltipId" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)